ui/components/multichain/detected-token-banner/detected-token-banner.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { renderWithProvider, screen, fireEvent } from '../../../../test/jest';
+import configureStore from '../../../store/store';
+import testData from '../../../../.storybook/test-data';
+
+import { DetectedTokensBanner } from './detected-token-banner';
+
+describe('DetectedTokensBanner', () => {
+  let setShowDetectedTokensSpy;
+  const args = {};
+
+  beforeEach(() => {
+    setShowDetectedTokensSpy = jest.fn();
+    args.setShowDetectedTokens = setShowDetectedTokensSpy;
+  });
+
+  it('should render correctly', () => {
+    const store = configureStore(testData);
+    const { getByTestId, container } = renderWithProvider(
+      <DetectedTokensBanner {...args} />,
+      store,
+    );
+
+    expect(getByTestId('detected-token-banner')).toBeDefined();
+    expect(container).toMatchSnapshot();
+  });
+  it('should render number of tokens detected link', () => {
+    const store = configureStore(testData);
+    renderWithProvider(<DetectedTokensBanner {...args} />, store);
+
+    expect(
+      screen.getByText('3 new tokens found in this account'),
+    ).toBeInTheDocument();
+
+    fireEvent.click(screen.getByText('Import tokens'));
+    expect(setShowDetectedTokensSpy).toHaveBeenCalled();
+  });
+});
```

Comment:
Could improve coverage slightly by running a test for this line of code
```
yarn jest ui/components/multichain --coverage --coverageReporters='html'
```
Then look in `coverage/` folder and run the index.html file in your browser

<img width="754" alt="Screenshot 2023-03-10 at 12 08 15 PM" src="https://user-images.githubusercontent.com/8112138/224417952-e3f1d7f0-fa32-4250-9b74-b02257592283.png">